company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise1Test.java

```diff
@@ -60,7 +61,8 @@ public void getCustomerCities()
     @Test
     public void getLondonCustomers()
     {
-        MutableList<Customer> customersFromLondon = null; // this.company.getCustomers()...
+        Predicates<Customer> predicate = Predicates.attributeEqual(Customer::getCity, "London");
+        MutableList<Customer> customersFromLondon = this.company.getCustomers().select(predicate);
```

Comment:
This certainly works, but I'd prefer to have the solutions use a lambda here.

Reply:
Changed it now :)